src/mocking/server/data.ts

```diff
@@ -65,12 +78,16 @@ export const createMockData = (server: MirageJSServer) => {
       media: videoMedias[mediaIndex],
       license: licenseEntities[idx % licenseEntities.length],
     })
+  })
 
-    server.create('VideoViewsInfo', {
-      id: video.id,
-      views: video.views,
+  videos
+    .filter((_, idx) => FEATURED_VIDEOS_INDEXES.includes(idx))
+    .map((video) => {
```

Comment:
I guess this `map` was here when you returned the value previously, I think now `forEach` will work fine?